### PR TITLE
Prepare for release

### PR DESCRIPTION
**Pull request by psafont**
Add license file with the ISC license with the date of its upload. Fill missing notes in the changelog.
Update to cmdliner 1.1.0 for the tests, do not fail when xenstore socket is not available to avoid failing needlessly on CI. Update opam and dune metadata.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Copyright 2017 Citrix
```
Comment (edwintorok) [inline LICENSE]: If we're adding a file anyway we might as well add the new copyright headers with Cloud Software Group

Comment (edwintorok) [reply LICENSE]: See [the SDK change recently merged into XAPI for an example ](https://github.com/xapi-project/xen-api/commit/79c5ec26d5f639fbb556f9071d242f3e425bc577#diff-af5fdcef76e2a64789240afc2fc623478436450488ee232210a85f1d716515d9)

Comment (edwintorok) [reply LICENSE]: 'Copyright (c) Cloud Software Group, Inc.'

File: lib_test/main.ml [hunk 2]
```diff
@@ -22,11 +30,8 @@ let cmd =
...
   ] in
-  Term.(ret (const test $ socket)),
-  Term.info "test_ezxenstore" ~doc ~man
-
-let () = match Term.eval cmd with `Ok x -> exit x | _ -> exit 1
-
-
-
+  Cmd.v
+    (Cmd.info "test_ezxenstore" ~doc ~man)
```
Comment (lindig) [inline lib_test/main.ml:34]: Is this the name of the binary? The `dune` files does not contain a `public_name`

**Review by lindig — APPROVED** [review]

**Comment by lindig** [comment]
Can we progress this?

**Review by robhoes — APPROVED** [review]

**Pull request merged**